src/ScreenshotWindow.vala

```diff
@@ -270,57 +265,36 @@ public class Screenshot.ScreenshotWindow : Hdy.ApplicationWindow {
         } while (File.new_for_path (full_file_name).query_exists ());
 
         screenshot.save (full_file_name, format);
+
+        var file_icon = new FileIcon (File.new_for_path (full_file_name));
```

Comment:
I am not getting a notification when running as Flatpak although it works OK native.  It seems to be linked to using a FileIcon (or at least, this one).  Using a ThemedIcon works OK in Flatpak.